src/backend/distributed/planner/distributed_planner.c

```diff
@@ -66,14 +66,20 @@ static uint64 NextPlanId = 1;
 /* keep track of planner call stack levels */
 int PlannerLevel = 0;
 
+typedef struct DistributedStmtOptions
```

Comment:
Options doesn't seem like the right name since it captures the whole distributed statement (twice!). Maybe `DistributedPlanningContext` or ..?